e2e/linter/src/linter.test.ts

```diff
@@ -506,6 +506,76 @@ export function tslibC(): string {
       expect(libEslint.overrides[1].extends).toBeUndefined();
       expect(libEslint.overrides[1].extends).toBeUndefined();
     });
+
+    it('(Angular standalone) should set root project config to app and e2e app and migrate when another lib is added', () => {
```

Comment:
@meeroslav Standalone means something different in angular. (Standalone components and APIs etc) This would probably be better titled something like Angular root-project

Reply:
It's a test in the linter e2e and matches our preset name so I would leave this as is, but I'll rename the PR.